### ISSUE TITLE
NumberFormatException crash while using the library 

### ISSUE THREAD

**Issue by cohenlior** [10bis/CompactCreditInput#14]
Hi Gil,

We are getting: Fatal Exception: NumberFormatException: For input string: "/" in CreditCardDateMaskWatcher.kt, onInvalidContent(CreditCardDateMaskWatcher.kt:56)

I cannot reproduce it, but it is gaining momentum for some users.

Can you please help us and fix it? :)

Thanks in advance!

**Comment by gilgoldzweig** [comment]
It happens when dealing with pasting dates. If I'm not wrong. I'm not sure If I currently have time to fix it, but it should a relatively simple fix

**Comment by cohenlior** [comment]
Yes, you are probably right for the reason and of course for the fix. will it be helpful if we open a pull request?

**Comment by gilgoldzweig** [comment]
By all means, it shouldn't be too difficult, I just don't have time to deal with it at the moment
